CLAUDE.md

```diff
@@ -0,0 +1,23 @@
+# PocketSizeFund Assistant Memory
+
+## GitHub Issue Creation
+
+When asked to create a new GitHub issue:
+
+1. Follow the structure in `.github/ISSUE_TEMPLATE.md`
+2. Include:
+   - A descriptive title (without [FEATURE|BUG] prefix)
+   - Add the "backlog" label and either "bug" or "feature" label as appropriate
```

Comment:
I'm not super inclined to have "ready" or "backlog" labels since we have status columns in the kanban board. Either we add columns there or (my personal preference) we just say everything is "To Do" and there's no need to differentiate.

Reply:
![CleanShot 2025-05-14 at 09 53 43](https://github.com/user-attachments/assets/8295c4e0-bcac-4d7c-8b7f-34adab9c4005)